### ISSUE TITLE
Thumbnail graphics in project carousels on the Home Page missing text descriptions

### ISSUE THREAD

**Issue by ChrisC** [LLK/scratch-www#7197]
### Expected Behavior

Every image should have a descriptive text alternative for screen reader users.

### Actual Behavior

All of the thumbnail graphics in the Project Carousels are missing alternative text descriptions. These images are user-generated, so the user should have a way to add alternative text to their project thumbnails to address this. There may also be a way to add a generic descriptive text that indicates to a screen reader user that the image is an user-generated thumbnail for the project. 

### System Details
Browser: Firefox, Screen Readers: Jaws/NVDA
